reco_utils/recommender/vowpal_wabbit/__init__.py

```diff
@@ -0,0 +1,8 @@
+__title__ = "Microsoft Recommenders"
```

Comment:
we don't need this info here, I guess you copied this from the root folder of recommenders

>

Reply:
oops yep